chains/evm/cli/utils/utils.go

```diff
@@ -25,25 +23,6 @@ func init() {
 	UtilsCmd.AddCommand(hashListCmd)
 }
 
-type EventSig string
-
-func (es EventSig) GetTopic() common.Hash {
-	return crypto.Keccak256Hash([]byte(es))
-}
-
-//
-//func IsActive(status uint8) bool {
-//	return ProposalStatus(status) == Active
-//}
-//
-//func IsPassed(status uint8) bool {
-//	return ProposalStatus(status) == Passed
-//}
-//
-//func IsExecuted(status uint8) bool {
-//	return ProposalStatus(status) == Executed
-//}
-
 // UserAmountToWei converts decimal user friendly representation of token amount to 'Wei' representation with provided amount of decimal places
 // eg UserAmountToWei(1, 5) => 100000
 func UserAmountToWei(amount string, decimal *big.Int) (*big.Int, error) {
```

Comment:
Could you also move these functions to some separate file - maybe together with ` WaitForProposalExecuted`, as they don't make any sense inside this file.